altsrc/yaml_file_loader.go

```diff
@@ -78,6 +80,11 @@ func loadDataFrom(filePath string) ([]byte, error) {
 			return nil, fmt.Errorf("Cannot read from file: '%s' because it does not exist.", filePath)
 		}
 		return ioutil.ReadFile(filePath)
+	} else if runtime.GOOS == "windows" && strings.Contains(u.String(), "\\") {
```

Comment:
At least on the machines I could try, `u.Path` is empty on Windows, so the previous if does not apply and falls onto the error on the last condition. It gets detected if you prefix with file:// but that seems not friendly.